build.sbt

```diff
@@ -39,13 +58,25 @@ lazy val generators = Project(
   AutomateHeaderPlugin
 )
 
+lazy val triplesStoreClient = Project(
+  id = "triples-store-client",
+  base = file("triples-store-client")
+).settings(
+  commonSettings
+).dependsOn(
+  generators % "test->test"
+).enablePlugins(
+  AutomateHeaderPlugin
+)
+
 lazy val tinyTypes = Project(
   id = "tiny-types",
   base = file("tiny-types")
 ).settings(
   commonSettings
 ).dependsOn(
-  generators % "test->test"
+  triplesStoreClient % "compile->compile",
```

Comment:
sbt can combine the scope mappings in a single string like `% "compile->compile; test->test"` - but totally fine like this as well, ofc

Reply:
Yes, I didn't know you can do it like this but saw it on your PR. I'll change it here.